version.properties

```
@@ -1,8 +1,8 @@
 #Mon, 30 Aug 2021 11:48:28 +0200
 
 # Version Information for Vanilla / Release builds
-versionName=19.1
-versionCode=1165
+versionName=19.1.1
+versionCode=1167
```

1167 versionCode is already used for alpha-342 on PlayStore.
```suggestion
versionCode=1168
```

---

I even looked at [the file](https://github.com/wordpress-mobile/WordPress-Android/blob/release/19.2/version.properties) and missed the alpha versions. @mokagio Sorry for the incorrect approval 🤦

---

I looked at the file too to see which number to increase by one 🤦‍♂️ 

🙃 _At least I tried..._